<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorPage</class>
 <widget class="QWidget" name="CalculatorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelLastUpdate">
     <property name="text">
      <string>Update status</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Currency exchange calculator</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="editSrcAmount">
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxSrcCurrency">
       <property name="insertPolicy">
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>worth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxResultCurrency">
       <property name="insertPolicy">
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelResultAmount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>??</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblBtcToUsd">
     <property name="text">
      <string notr="true">1 BTC = ?? USD (Bitstamp)</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionUpdateResult">
   <property name="text">
    <string>UpdateResult</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbxResultCurrency</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionUpdateResult</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxSrcCurrency</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionUpdateResult</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editSrcAmount</sender>
   <signal>valueChanged(double)</signal>
   <receiver>actionUpdateResult</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
